docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,649 @@
+allows you to navigate around your computer. 
+
+Format:
+```shell script
+cd <directory>
+```
+
+- You can chain multiple directories together by adding '/' between each directory if you would like
+to jump to a specific one.
+- You can jump up directories with the '..' notation.
+- You can jump to your home directory (the one with Desktop, Documents, ... inside of it) by just typing `cd` or
+`cd ~/`
+
+Examples:
+```shell script
+# Go to your desktop
+cd Desktop
+
+# Go to back to your root directory
+cd ..
+
+# Go up multiple directories
+cd ../../../
+
+# Go back to your root directory
+cd ~/
+```
+
+#### ls
+
+`ls` is also a very commonly used commands. It lists all of the contents of a directory (both files and 
+sub-directories).
+
+Format:
+```shell script
+ls <optional filepath>
+```
+
+- You can list hidden files in a directory with the `-a` flag
+
+Examples:
+```shell script
+# List files in the current directory
+ls
+
+# List all files in the current directory
+ls -a
+
+# List files in the Desktop directory
+ls Desktop
+```
+
+#### pwd
+
+`pwd` stands for 'print working directory', and it logs the current directory you're located at.
+
+Format:
+```shell script
+pwd
+```
+
+#### mv
+
+`mv` is a command that lets you move a file or directory to another location. It can also be used
+to rename a file or directory (since that's the same thing as moving it in a way).
+
+Format:
+```shell script
+mv <from filepath> <to filepath>
+```
+
+- You can't move a file inside of itself or one of its subdirectories (without messing 
+around with symbolic links, but that's way outside of the material of this course).
+
+Example:
+```shell script
+# Move the README.md file to the parent directory without renaming it
+mv README.md ../
+
+# Rename the test.txt file to test.md
+mv test.txt test.md
+
+# Move the test.txt file to the Desktop from Documents and rename it to hello.world
+mv Documents/test.txt Desktop/hello.world
+```
+
+#### cp
+
+`cp` copies a file, and it's very similar to `mv`. 
+
+Format:
+```shell script
+cp <exiting file or directory> <new file or directory>
+```
+
+- If you're copying a directory, you need to make sure to include the `-r` flag so that it 
+copies everything in the directory too.
+
+Examples:
+```shell script
+# Copy test.txt to Desktop
+cp test.txt Desktop
+
+# Copy the Desktop directory to Documents 
+cp Desktop Documents -r
+```
+
+#### mkdir
+
+`mkdir` creates a directory.
+
+Format:
+```shell script
+mkdir <directory name>
+```
+
+Examples:
+```shell script
+# Create a Desktop2 directory
+mkdir Desktop2
+```
+
+#### rm
+
+`rm` deletes a file or a directory.
+
+```shell script
+rm <path to file or directory>
+```
+
+- Be careful using this command. Once it's used, files are pretty much unrecoverable without special software.
+- If you're deleting a directory, you'll need to pass in the `-rf` flags.
+    - The `-r` flag makes it recursive, it will delete all subdirectories.
```

Comment:
sometimes need sudo to delete directories

Reply:
Wait which directories? Unless you're in a directory you don't technically own (like /ext or something), it shouldn't require you to do that.

Reply:
Probably fine omitting here. Maybe have a section on `sudo` if we really want to but not necessary imo